test/frames/frames_helper_test.rb

```diff
@@ -21,6 +21,26 @@ class Turbo::FramesHelperTest < ActionView::TestCase
     assert_dom_equal %(<turbo-frame id="message_1"></turbo-frame>), turbo_frame_tag(record)
   end
 
+  test "frame with invalid argument should raise ArgumentError" do
```

Comment:
The record variables could be ignored here and pass the argument directly. I don't see much value in declaring them